packages/ai-jsx/src/core/conversation.tsx

```diff
@@ -394,6 +406,20 @@ export async function ShrinkConversation(
       stop: (e) => isConversationalComponent(e) || e.tag === InternalShrinkable,
     });
 
+    const invalidChildren = rendered.filter((el) => typeof el === 'string' && el.trim()) as string[];
```

Comment:
@petersalas I found that I needed to repeat the check here – can you think of anywhere else I should do it?

Reply:
Sigh, yeah. I don't think so -- this mixed conversation + shrinkable thing felt icky when I did it because I don't think I did it previously.